src/ui/src/app/user/check-user/check-user.component.ts

```diff
@@ -33,16 +33,16 @@ export class CheckUserComponent {
     ;
     if(this.userForm.valid){
      
-      this.userService.getEmployeeSignUp(this.userForm.value.employeeCode,this.userForm.value.rfId).subscribe({
+      this.userService.getEmployeeSignUp(this.userForm.value.employeeCode,this.userForm.value.rfid).subscribe({
```

Comment:
camelCase, there is a issue yesterday , so please correct on the form also

Reply:
done